src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -625,15 +708,20 @@ private void assignTask(TestVariables taskVariables) {
             .statusCode(HttpStatus.NO_CONTENT.value());
 
         assertions.taskVariableWasUpdated(taskVariables.getProcessInstanceId(), "taskState", "assigned");
-        assertions.taskStateWasUpdatedInDatabase(
-            taskVariables.getTaskId(), "assigned", caseworkerCredentials.getHeaders()
-        );
-        assertions.taskFieldWasUpdatedInDatabase(
-            taskVariables.getTaskId(), "assignee", assigneeId, caseworkerCredentials.getHeaders()
+
+        //This need to be changed when assign endpoint story RWA-1582 is played
+        Map<String, Matcher<?>> taskValueMap = Map.of(
```

Comment:
again

Reply:
I think we will connect and discuss regarding the best approach to deal with these?

Reply:
done